### PR TITLE
Add Preview Snippet & Character Counter support for text tab of Classic Editor

### PR DESCRIPTION
**Pull request by arnaudbroes**
Issue #3052

## Proposed changes

Adds support for the Preview Snippet and Character Counter for the text tab of the Classic Editor.

## Types of changes

- Bugfix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Improves existing functionality
- Improves existing code

## Checklist

- [x] I've selected the appropriate branch (ie x.y rather than master).
- [x] I've created an appropriate title, descriptive of what the PR does.
- [ ] Travis-ci runs with no errors.

## Testing instructions

1. Reproduce the issue here - https://github.com/semperfiwebdesign/all-in-one-seo-pack/issues/3052
2. Install the PR and confirm that it fixes the issue.
3. Do a quick check of the Preview Snippet and Character Counter for the Edit post page (both for the Gutenberg Editor and Classic Editor). 
4. Do a quick check of the Character Counter for the Home Title and Home Description fields in the General Settings and Social Meta menu.

## Further comments

Grunt will fail on this PR and that's fine.

I've also deleted the relevant minified files for now since we're not using them at the moment and now are out-of-date.

Refactored some of the code too and created a shared admin JS file for general functions.

### CODE REVIEW

**Comment by arnaudbroes** [comment]
@wpsmort when in the text tab (HTML editor) of the Classic Editor, the Preview Snippet and Character Counter will only update once you leave/defocus the editor (basically when you click anywhere else with your mouse). I've opened an issue here to see if I can improve this - https://github.com/semperfiwebdesign/all-in-one-seo-pack/issues/3060, but this should be sufficient for now.

**Comment by wpsmort** [comment]
This is tested and is good.

File: js/admin/aioseop-admin-functions.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
+/**
```
Comment (EkoJR) [inline js/admin/aioseop-admin-functions.js]: Tabs instead of spaces for indenting.

Comment (arnaudbroes) [reply js/admin/aioseop-admin-functions.js]: @EkoJR fortunately Visual Studio Code makes it really easy.

image

**Comment by arnaudbroes** [comment]
@EkoJR this can be reviewed again.

**Comment by arnaudbroes** [comment]
@EkoJR forgot to run Grunt. This should be good now and can be reviewed again .

**Pull request merged**